### DISCUSSION TITLE
ETS2 Convoy Server Setup Issue with App ID

### DISCUSSION THREAD

**Discussion by derBerliner1157** [LsHallo/ets2-dedicated-convoy-server#30] [Ideas]
Hello everyone,

I recently set up an ETS2 convoy server using the provided Docker image—thanks to everyone involved in creating it! The server is running smoothly now, but I encountered some challenges during the setup process that I wanted to share.

Initially, I followed the setup instructions closely, but the server didn’t work as expected. After a day of troubleshooting without success, I decided to rebuild the container from scratch. That’s when I noticed something interesting: the App ID had changed from 227300 to 1948160.

To test this, I manually set the App ID back to 227300. However, this resulted in an error where the server couldn’t locate all the required files. When I reverted the App ID to 1948160 and generated a new Logon Token for this ID, the server started working perfectly.

As someone fairly new to this topic, I’m not entirely sure why this happened. For context, I’m using Docker on Kali Linux and managing the container configuration through Portainer using environment variables.

I wanted to bring attention to this issue in case others run into the same problem. Aside from this hiccup, the server is running flawlessly.

**Comment by LsHallo** [comment]
I think you are a little confused with the AppIDs.
The server for ETS2 has AppID 1948160. This is used to download the server files in der Dockerfile. 
ETS2 itself is AppID 227300. The official readme for the dedicated server instructs you to create a token with AppID 227300 to use with the server.
I've just created a new token for 227300 and started the server successfully. No image rebuild needed.
I then created a new token with 1948160 as the AppID. This test did not log in successfully:
```
ets2  | [MP] LogOn to persistent server account.
ets2  | *** ERROR *** : [MP] Error: Steam log on failed - code 15 .
```

So my README advice stands. Use 227300 for the token. 1948160 will not work.
AppID 227300 is not used in the container and that is correct.

The only thing is, that you need to own ETS2 for it to work.

**Reply by derBerliner1157** [reply]
Thank you for the clarification regarding the App IDs.

At the moment, I only have one active Logon Token, and it is **not** for the 227300 App ID. Interestingly, I did manage to get the server working once with a Logon Token for 227300. However, after making further changes to the environment configuration, it stopped working.

I’m not entirely sure what caused this issue, but I thought sharing my experience might be helpful for others who might encounter similar problems during setup.
